src/platform-aws.c

```diff
@@ -583,8 +504,9 @@ int platform_init(const char **provider_filter)
 		}
 
 		NCCL_OFI_INFO(NCCL_INIT | NCCL_NET,
-				"Running on %s platform, Setting NCCL_TOPO_FILE environment variable to %s",
-				get_platform_type(), topology_path);
+		              "Running on %s platform, Setting NCCL_TOPO_FILE environment variable to %s",
```

Comment:
Why does git think this line changed?

Reply:
My .clangformat detected the change in the line and tried to enforce the formatting. I will correct it.